syft/frameworks/crypten/hook/hook.py

```diff
@@ -5,29 +5,57 @@
 import crypten
 import torch as th
 
-methods_to_hook = ["load"]
+
+class CrypTenPlanBuild(object):
+    @staticmethod
+    def f_return_none(*args, **kwargs):
+        return None
+
+    @staticmethod
+    def f_return_cryptensor(*args, **kwargs):
+        return crypten.cryptensor(th.zeros([]))
+
+    @staticmethod
+    def f_return_model_or_cryptensor(*args, **kwargs):
+        if args[0] == "crypten_model":
+            return crypten.nn.Module()
+        else:
+            return crypten.cryptensor(th.zeros([]))
+
+
+crypten_plan_hook = {
+    crypten: {"load": CrypTenPlanBuild.f_return_model_or_cryptensor,},
```

Comment:
According to Ayoub suggestion this would also be a bit cleaner

Reply:
This ```load``` is used only to create the "shell" for the built plan.
Also, currently, I removed the approach with OnnxModel.